### DISCUSSION TITLE
implementing bio-metric

### DISCUSSION THREAD

**Discussion by maifeeulasad** [Spikeysanju/Expenso#58] [Show and tell]
as per #52 

I've started implementing bio-metric in my fork, in a specific feature branch.

Current progress can be seen here : https://github.com/maifeeulasad/Expenso/tree/0f4ddfab7244626677f3b39a6871b9df0e793f9c

Using `androidx.biometric:biometric:1.2.0-alpha03`

Now need to create to a settings menu, to enable/disable this feature

**Comment by maifeeulasad** [comment]
@Spikeysanju how should i store this(this feature enabled or disabled) data.

Should I simply use sharedpref or create repo(i think this will be unnecessary)?

**Reply by Spikeysanju** [reply]
@maifeeulasad  Shall we use `DataStore` for this?.. Currently we're already using `DataStore` for storing `Day/Night` Mode.

I think that will be perfect for this feature. Feel free to leave your thoughts on this 👍

**Reply by maifeeulasad** [reply]
@Spikeysanju I've used `DataStore` and tried to follow the project structure as much as possible. 

And here is the pull : #59 

Let me know if it needs any modifications or something.
